public class App {
    public static void main(String[] args) throws Exception {
        new Exerc1().Run(); //Ele meio que chama os dois
    }
}
